build.gradle

```diff
@@ -119,10 +119,10 @@ publishing {
 
 ext {
     kafkaVersion = "2.8.1"
-    slf4jVersion = "1.7.36"
+    slf4jVersion = "2.0.7"
```

Comment:
OpenSearch (and I believe Apache Kafka) still use 1.7.x line

Reply:
yes you are right will close PR